engines/account/app/mailers/account/signup_mailer.rb

```diff
@@ -13,11 +13,9 @@ def confirm_email(signup)
 
     private
 
-    # TODO Find a way to manage express URLs
-    #
     # @return [String]
     def confirm_url
-      "http://10.0.3.70:8000/signup/complete/#{@signup.token}"
+      "#{::Shared::PublicUrl.base_url}/signup/complete/#{@signup.token}"
```

Comment:
You could also address front end url generation know. I mean this kind of hardcode paths `/signup/complete/:token`